src/otx/algo/detection/atss.py

```diff
@@ -313,15 +155,13 @@ def _build_model(self, num_classes: int) -> SingleStageDetector:
             "pos_weight": -1,
             "debug": False,
         }
-        test_cfg = DictConfig(
-            {
-                "nms": {"type": "nms", "iou_threshold": 0.6},
-                "min_bbox_size": 0,
-                "score_thr": 0.05,
-                "max_per_img": 100,
-                "nms_pre": 1000,
-            },
-        )
+        test_cfg = {
+            "nms": {"type": "nms", "iou_threshold": 0.6},
+            "min_bbox_size": 0,
+            "score_thr": 0.05,
+            "max_per_img": 100,
+            "nms_pre": 1000,
+        }
```

Comment:
My guess is that we'll need to refactor `train_cfg` and `test_cfg` (+other cfg) appropriately in the future as well.

Reply:
I think it should be configurable as well as "image_size", "mean", "std", etc.

Reply:
I agree with your opinion, but if doing it, the number of arguments will be so many.
Let's talk about this for the next topic.
Before this, I organized arguments that are same with the default value (https://github.com/openvinotoolkit/training_extensions/pull/3636/commits/c04aa58ccb4122de61bdf3561f7c76493114ecd7).